### DISCUSSION TITLE
Task 01 Lab 02.03

### DISCUSSION THREAD

**Discussion by rsbgonzalez** [rramosp/ai4eng.v1#20] [02 - PYTHON]
Profe Raúl buenas tardes. Estoy tratando de subir la solución a la task 01 del lab 02.03 pero me aparece este error:

![image](https://user-images.githubusercontent.com/85143626/120710162-6d649400-c483-11eb-87d9-c5491f6367ab.png)

Y la verdad no tengo idea de que se trata.

Me puede ayudar por favor?

**Comment by rramosp** [comment]
fíjate que las funciones que tienes que hacer tienen un parámetro (`df`) .... es sobre el dataframe `df` que tienes que implementar tu función ... probablemente estés usando la variable `d`  dentro de la función, y no `df` ... el autocorrector se ejecuta en un entorno independiente del notebook para asegurar el encriptado y las variables globales del notebook no están visibles cuando tu función la ejecuta el corrector automático

**Reply by rsbgonzalez** [reply]
listo profe!, solucionado.

Me ayuda con la otra duda porfa, la del closest scalar.
